docs/datatypes/tset.md

```diff
@@ -0,0 +1,251 @@
+---
+id: datatypes_tset
+title: "TSet"
+---
+
+A `TSet[A]` is transactional data structure built on top of `TMap[A]`.
```

Comment:
I would rather describe it as a mutable set that can participate in transactions. See `tref.md` as an inspiration.